src/components/UI/modal/SmallCard.js

```diff
@@ -22,6 +30,7 @@ const SmallCard = (props) => {
                 <div className="product-price">{price} грн</div>
                 <div
                     className="remove-item"
+                    onClick={ () => deleteFromBasket(props) }
```

Comment:
No need to create separated function here. Just add `removeProductFromBasket(props)` instead of  `deleteFromBasket(props)`. And delete `deleteFromBasket`(15 line of code).